src/client/transaction-lifecycle-hooks-request-handler.ts

```diff
@@ -0,0 +1,69 @@
+import { HttpClient } from './http-client';
+import * as Parsiq from './parsiq-client';
+import { AxiosRequestConfig, isAxiosError } from 'axios';
+import { IAxiosRetryConfig } from 'axios-retry';
+import { TSUNAMI_BASE_URL } from './urls';
+import { TsunamiError } from './tsunami-error';
+import { CreateTransactionLifecycle, TransactionLifecycleData } from '../dto/transaction-lifecycle';
+
+export class TransactionLifecycleHooksRequestHandler extends HttpClient {
+  constructor(
+    apiKey: string,
+    chain: Parsiq.ChainId,
+    config: { axiosConfig?: AxiosRequestConfig; retryConfig?: IAxiosRetryConfig } = {
+      axiosConfig: {},
+      retryConfig: {},
+    },
+  ) {
+    const { axiosConfig = {}, retryConfig = {} } = config;
+    super(TSUNAMI_BASE_URL, chain, apiKey, axiosConfig, retryConfig);
+  }
+
+  public async createTxLcHook(createHook: CreateTransactionLifecycle): Promise<string> {
```

Comment:
why `createTxLcHook` (etc), if it's already in the TransactionLifecycle namespace?

Reply:
i guess it's ±irrelevant, since in the public interface it's without redundancy

Reply:
not needed strictly, but other handlers do have specifics, so rather to be on par with the rest than for necessity

Reply:
yeah, sorry - now i see difference between public interface and request handlers